tests/integration/conftest.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+import pytest
+from lightkube.resources.core_v1 import Namespace
+
+from spark8t.domain import Defaults
+from spark8t.services import K8sServiceAccountRegistry, KubeInterface, LightKube
+
+integration_test_flag = bool(int(os.environ.get("IE_TEST", "0")))
+
+
+@pytest.fixture(autouse=True)
+def integration_test():
+    if not integration_test_flag:
+        pytest.skip(
+            reason="Integration test, to be skipped when running unittests",
+        )
```

Comment:
I'm unsure about what was the purpose of this originally.
It was set for part of the integration tests, but not all.

Now this implementation will set it for all (but of course that's easy to change).

However I'm curious about the purpose of this one, and why we chose this solution (and not a custom pytest marker or alike)?

Reply:
The idea of this flag was to enable integration tests (that requires external component such as microk8s to be running) on demand. 

The idea is that we annotate the tests that requires external components with a decorator that depending on an env variable (but with pytest we could instead also use a CLI argument) we skip these. This allow us to also have all metrics (reports, coverage) with and without integrations in an easy way

Reply:
Excellent, thank you.

I've adjusted the usage accordingly

Reply:
Could we just use a `pytest.mark` decorator here on relevant functions, then run tests with that mark flag when we want to skip them?